src/ru/skypro/Book.java

```diff
@@ -0,0 +1,58 @@
+package ru.skypro;
+
+import java.util.Objects;
+
+public class Book {
+    private Author author;
+    private String titleBook;
```

Comment:
```suggestion
    private final Author author;
    private final String titleBook;
```
поля можно пометить модификтором final, поскольку они у нас неизменяемые по коду: мы единожды задаем им значение в конструкторе и более не меняем. Данный модификатор необязательно задавать, если мы и так уверены по коду, что значение не поменяется, но с модификатором это выглядит более явно как программисту, так и для компилятора (он будет ругаться при попытке присвоить новое значение)